tests/app/celery/test_process_ses_receipts_tasks.py

```diff
@@ -102,20 +102,32 @@ def test_ses_callback_should_call_send_delivery_status_to_service(
     )
 
 
-def test_ses_callback_should_send_statsd_statistics(mocker, client, sample_template, sample_notification):
+def test_wt_ses_callback_should_log_total_time(
+    mocker,
+    client,
+    sample_template,
+    sample_notification,
+):
     template = sample_template(template_type=EMAIL_TYPE)
     with freeze_time('2001-01-01T12:00:00'):
```

Comment:
Did they even get text messages back in 2001 hahah

Reply:
I had a palm pilot back in 2001 ;) and blackberry was a thing.

Reply:
haha, throughout the tests they used this to avoid overlapping notifications while the workers ran in parallel. It works for some tests and not for others.